src/index.js

```diff
@@ -31,7 +31,7 @@ function startElectron (config, ports) {
         args = debugPortsArgs.concat(extraArgs);
     }
 
-    spawn(cmd, args, { stdio: 'ignore' });
+    spawn(cmd, args, { stdio: process.env.DEBUG ? 'inherit' : 'ignore' });
```

Comment:
The `DEBUG` environment variable has some special meaning for many Node.js since it controls features of the [debug](https://www.npmjs.com/package/debug) module. The `testcafe` module uses it as well. I would like to use `pipe` in the `stdio` configuration and always send the piped output to the `debug` module. Something like:
```js
import debug from 'debug';

const debugLogger = debug('testcafe-browser-provider-electron');
const stdioLogger = debugLogger.extend('spawn').extend('stdio');

...

const process = spawn(cmd, args, { stdio: ['ignore', 'pipe', 'pipe'] });

for (const stream of [process.stdout, process.stderr])
    stream.on('data', data => stdioLogger(String(data)));

Reply:
Willfix. I’ll add and configure debug